pkg/device/device-util.go

```diff
@@ -177,13 +193,37 @@ func getAllPartsUsed(diskMetaName string, partitionName string) ([]partUsed, err
 		}
 		for _, tmp := range tmpList {
 			if tmp[len(tmp)-1] == partitionName {
-				partNum, _ := strconv.ParseInt(tmp[0], 10, 32)
-				pList = append(pList, partUsed{disk.DiskName, uint32(partNum)})
+				partUsed, err := parsePartUsed(disk.DiskName, tmp)
+				if err != nil {
+					return pList, err
+				}
+				pList = append(pList, partUsed)
 			}
 		}
 	}
 	return pList, nil
+}
 
+func parsePartUsed(diskName string, row []string) (PartUsed, error) {
```

Comment:
Can you also add unit tests to this function, since we are doing many conversions and string trimming, need to make sure it does not break anything,

Reply:
done.